README.md

```diff
@@ -1,5 +1,7 @@
 # README
 
+-- first pr
+
 This README would normally document whatever steps are necessary to get the
 application up and running.
```

Comment:
@pear-on explain this diff

Reply:
The diff represents a change in the README file of this repository. Two lines have been added (`+-- first pr
+
`). This indicates that a description or note about the first PR has been added to the README. Let me know if you need further explanation.